Automattic/Gravatar-SDK-Android
# Update the style of the progress indicator when uploading a new Avatar

Closes #436 

### Description

It was reported that the progress bar could be easily missed, so I'm updating the color and the size of the progress indicator.

The color is hardcoded as it's displayed on an overlay with a hardcoded color as well.  

| Before | After |
|---|---|
| ![progress_before](https://github.com/user-attachments/assets/90e695e1-5015-4f30-aac7-3bfbb7b95f32) | ![progress_after](https://github.com/user-attachments/assets/4a719827-3f44-45f6-8935-20b090c1f874) |

### Testing Steps

1. Open the QE
2. Log in
3. Upload a new photo
4. Confirm the progress indicator matches the above GIF
